### ISSUE TITLE
Improve Heroku Downtime Handling

### ISSUE THREAD

**Issue by dahlbyk** [huboard/huboard-web#236]
Following up on [today's downtime](https://github.com/huboard/huboard-web/issues/235), two observations:

1. We should implement Heroku [customer error/maintenance pages](https://devcenter.heroku.com/articles/error-pages#customize-pages). My first thought is to host the pages from this repo with GitHub Pages.
2. We need to have some sort of uptime monitor, ideally integrated with StatusPage (e.g. http://doers.statuspage.io/public-metrics/pingdom/ or http://doers.statuspage.io/public-metrics/new-relic/). For now I've set up huboard.com and huboard.com/huboard pings with NewRelic Synthetics.

**Comment by discorick** [comment]
> We should implement Heroku customer error/maintenance pages. My first thought is to host the pages from this repo with GitHub Pages. Heroku pulls our page in with an iframe.

Completed via cloudfront: 

503: https://d1e976wof7qcp5.cloudfront.net/static/503.html
Maintenance: https://d1e976wof7qcp5.cloudfront.net/static/maintenance.html

**Comment by discorick** [comment]
> We need to have some sort of uptime monitor, ideally integrated with StatusPage (e.g. http://doers.statuspage.io/public-metrics/pingdom/ or http://doers.statuspage.io/public-metrics/new-relic/). For now I've set up huboard.com and huboard.com/huboard pings with NewRelic Synthetics.

Would you consider this finished as well @dahlbyk ? We have pretty good coverage on status.huboard.com

**Comment by dahlbyk** [comment]
> Completed via cloudfront:
> 
> 503: https://d1e976wof7qcp5.cloudfront.net/static/503.html
> Maintenance: https://d1e976wof7qcp5.cloudfront.net/static/maintenance.html

Confirmed! :100:

> > We need to have some sort of uptime monitor, ideally integrated with StatusPage (e.g. http://doers.statuspage.io/public-metrics/pingdom/ or http://doers.statuspage.io/public-metrics/new-relic/). For now I've set up huboard.com and huboard.com/huboard pings with NewRelic Synthetics.
> 
> Would you consider this finished as well @dahlbyk ? We have pretty good coverage on status.huboard.com

We have pretty good coverage, but if we have a problem that our fault, rather than a vendor, we have to manually update. Since we're using NR, care to look into pushing some sort of metric into StatusPage?

**Comment by discorick** [comment]
> We have pretty good coverage, but if we have a problem that our fault, rather than a vendor, we have to manually update. Since we're using NR, care to look into pushing some sort of metric into StatusPage?

Yep worth looking into. Also NR didn't pick up our last outtage from the GH API which is a little concerning
